### ISSUE TITLE
Nightmare tag warning when substitution is used

### ISSUE THREAD

**Issue by RioT1133** [Naxesss/MapsetVerifier#35]
[this](https://riot1133.s-ul.eu/40grt1Nd) is what mv shows and [this](https://riot1133.s-ul.eu/CeY4Beby) is what my metadata is. i think the suggestion that should be displayed should be

"nightcore" in tags, consider "(Nightcore Mix)" instead of "(Sped Up Ver.)" in romanized title.

instead of the other way around since i already have "(Sped Up Ver.)" in the title

**Comment by Darius-Wattimena** [comment]
This is actually intended behaviour of the check. For songs that have `nightcore` in their tags, it's preferable to use the `(Nightcore Mix)` suffix. 

We should make a slight adjustment here so the warning is a bit more descriptive or even adjust this so it is not marked as a warning as the `(Sped Up Ver.)` title marking is correct.

**Comment by Darius-Wattimena** [comment]
> just to add here, the nightcore tag is usually added specifically when the title itself does not include nightcore, otherwise the tag would be obsolete

Fair point, I guess the approach for an informative message would be better, since people mainly go for `(Nightcore Mix)` nowadays, and suggest to add the following `sped up ver.` to tags instead.

> also i realised i wrote nightmare instead of nightcore oops

That was me LOL, been going over all issues so I am getting blind 😂

**Comment by hongaaronc** [comment]
<img width="1172" height="700" alt="Image" src="https://github.com/user-attachments/assets/529c95e1-1c62-42c9-a18c-e351383b6435" />

Seems to already be fixed from my testing. I assume Naxess probably did it already and it just needs to be released.
